packages/devtools_app/lib/src/shared/theme.dart

```diff
@@ -18,9 +18,9 @@ const contrastForegroundWhite = _contrastForegroundWhite;
 /// Constructs the light or dark theme for the app taking into account
 /// IDE-supplied theming.
 ThemeData themeFor({
-  @required bool isDarkTheme,
-  @required IdeTheme ideTheme,
-  ThemeData theme,
+  required bool isDarkTheme,
+  required IdeTheme? ideTheme,
```

Comment:
now IdeTheme should be able to be non-nullable everywhere

Reply:
done

Reply:
some tests are fixed by `setGlobal(IdeTheme, IdeTheme());`, but some are not
if test pumps widgets like this `await tester.pumpWidget(wrap(aboutDialog));`

themeFor gets null for ideTheme. it seems globals are not visible in this case.

reverted back to nullable

Reply:
fixed by updating 'wrap'